dpnp/dpnp_iface.py

```diff
@@ -252,6 +265,15 @@ def get_dpnp_descriptor(ext_obj,
     return False
 
 
+def get_execution_queue(qs):
```

Comment:
Input keywords and result needs to be clear described following the standard documentation format, since it will be visible in dpnp docs.
Or probably we need to move the new functions to `dpnp_algo_utils.pyx`.